src/Surfnet/StepupMiddleware/ManagementBundle/Controller/InstitutionConfigurationController.php

```diff
@@ -42,6 +44,25 @@
  */
 final class InstitutionConfigurationController extends Controller
 {
+    public function showAction()
+    {
+        $this->denyAccessUnlessGranted(['ROLE_MANAGEMENT']);
+
+        $response = array_map(
+            function (InstitutionConfigurationOptions $options) {
```

Comment:
how about implementing [this][1] (as is done in [about all others][2] :stuck_out_tongue_winking_eye: )

[1]: http://php.net/manual/en/class.jsonserializable.php
[2]: https://github.com/SURFnet/Stepup-Middleware/tree/develop/src/Surfnet/StepupMiddleware/ApiBundle/Identity/Entity